devtools/src/arcs-hot-reload.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {PolymerElement} from '../deps/@polymer/polymer/polymer-element.js';
+import {html} from '../deps/@polymer/polymer/lib/utils/html-tag.js';
+import {MessengerMixin} from './arcs-shared.js';
+
+class HotReload extends MessengerMixin(PolymerElement) {
```

Comment:
Add brief header documentation on what this is -- the logic it's providing a UI for, what components it interacts with, what behavior it manages.

Reply:
Added! :))